specification.md

```diff
@@ -95,7 +95,15 @@ An ingredient consists of
 An ingredient is represented as follows:
 
 1. The amount in italics
-2. Everything following is part of the name
+2. Everything following until the end of the line is part of the name.
+
+#### Name
+
+If a name contains a [link reference](https://spec.commonmark.org/0.28/#link-reference-definitions), the title of the link target recipe becomes the name of the ingredient.
```

Comment:
I think you mean an [inline link](https://spec.commonmark.org/0.28/#inline-link) here, as used in the example. 

Also in my opinion it would be better to use the [link text](https://spec.commonmark.org/0.28/#link-text) as the ingredients name. I'm not sure what to do about a link title, maybe just parse and serialise back as is?

Reply:
Yes, I mean inline-link and guess I got link-text and link-title confused.

Reply:
Plus, I am thoroughly confused by the "requested changes" UX of Gitlab.

Reply:
Should be resolved in 17653cdef52acadc6ab3023f177e8d19c68a5106